random-logs/filebeat.yml

```diff
@@ -0,0 +1,28 @@
+filebeat.inputs:
+- type: log
+  enabled: true
+  fields:
+    app_id: filebeat
+  paths:
+    - /usr/share/filebeat/random-logs/*.log
+
+filebeat.config.modules:
+  path: ${path.config}/modules.d/*.yml
+  reload.enabled: false
+
+# setup filebeat to send output to logstash
+output.logstash:
+  hosts: 
+    - "127.0.0.1:5045"
+
+# Write Filebeat own logs only to file to avoid catching them with itself in docker log files
+logging.level: error
+logging.to_files: false
+logging.to_syslog: false
+loggins.metrice.enabled: false
+logging.files:
+  path: /var/log/filebeat
+  name: filebeat
+  keepfiles: 7
+  permissions: 0644
+ssl.verification_mode: none
```

Comment:
Does this not conflict with the logging settings in the other filebeats container? I.e. wouldn't they overwrite each others' logs?